Gemfile

```diff
@@ -3,6 +3,7 @@ git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.6.3"
 
+gem "bcrypt", "3.1.13"
```

Comment:
gem này tác dụng gì nhỉ ?

Reply:
gem này có tác dụng mã hóa mật khẩu, cho phép lưu trữ mật khẩu của người dùng một cách an toàn